datalad_registry/templates/overview.html

```diff
@@ -40,16 +40,74 @@
 <div id="datalad-registry">
   <div class="content">
     <form action="{{ url_for('.overview') }}" formmethod="get">
-      <input type='search' name='filter'
+      <input id='search' type='search' name='filter'
           {%- if url_filter %} value="{{ url_filter|escape }}"
           {%- endif -%}
+             placeholder='Filter query'
       />
       <input type="hidden" name="sort" value="{{ sort_by }}"/>
       <input type='submit' value='Filter URLs'/>
-      {% if url_filter -%}
-        <a href="{{ url_for('.overview', sort=sort_by) }}"> remove filter</a>
+      {% if url_filter_error -%}
+        <p></p><span class="error">ERROR: {{ url_filter_error }}</span></p>
       {%- endif %}
     </form>
+    <button id="showQuerySyntax">Show filter query syntax</button>
+    {% if url_filter -%}
+        <a href="{{ url_for('.overview', sort=sort_by) }}"> remove filter</a>
+    {%- endif %}
+
+    <div id="QuerySyntaxModal" class="modal">
+
+    <!-- Modal content -->
+      <div class="modal-content">
+        <div class="modal-header">
+          <span class="close">&times;</span>
+          <h2>Filter Query Syntax</h2>
+        </div>
+        <div class="modal-body">
+          <p>It is WiP and could change. Meanwhile it allows for</p>
+          <ul>
+            <li>Simple singular <span class="content">WORD</span> (e.g., <span class="query">haxby</span>)
+              or a <span class="content">"QUOTED STRING"</span> (e.g., <span class="query">"James Haxby"</span>):
+              <ul>
+                <li>search is case-insensitive ATM so <span class="query">haxby</span> is identical to <span
+                        class="query">Haxby</span>;
+                </li>
+                <li>search is within the content, and not for entire words,
+                  so searching for <span class="query">ax</span> would find not only <span class="content">Haxby</span>
+                  but also <span class="content">tax</span> etc.
```

Comment:
```suggestion
                <li>Searching is performed within content rather than for entire words,
                  and so searching for <span class="query">ax</span> would find <span class="content">Haxby</span>,
                  <span class="content">tax</span>, etc.
```